server/src/servershot.js

```diff
@@ -856,6 +856,10 @@ Shot.upgradeSearch = function() {
             return resolve();
           }
           Shot.get("upgrade_search_only", rows[index].id).then((shot) => {
+            // This shouldn't really happen, but apparently can...
```

Comment:
Something weird in my local database, I don't know if it was due to testing or some latent bug I didn't notice before.